ibis/backends/snowflake/__init__.py

```diff
@@ -75,6 +78,7 @@ class SnowflakeExprTranslator(AlchemyExprTranslator):
 
 
 class SnowflakeCompiler(AlchemyCompiler):
+    cheap_in_memory_tables = _NATIVE_ARROW
```

Comment:
Can you add a test that checks that the backend is still running when this option is set to False?

Reply:
Tests in CI have this set to `False` currently. I am going to follow this up with a PR to add another job that makes sure this gets run when `_NATIVE_ARROW` is `True`.